### ISSUE TITLE
Suggestions / Improvements: Dark Skin

### ISSUE THREAD

**Issue by kersten-lohmeyer** [znuny/Znuny#584]
- Server OS: Debian 12
- Browser: Chrome (most current version)
- Znuny version: 7.1.2

First of all, thanks for a dark skin, now looking forward to a slim(mer) skin in dark and also light.

The gravatar colors are inverted in the dark skin. I think this is not intended:
<img width="193" alt="Bildschirmfoto 2024-07-25 um 19 58 44" src="https://github.com/user-attachments/assets/a6324da9-4d8b-48cd-a5a7-c8df60a5cdb2">

Also when you are on a page which is long enough for the scrollbar to appear, then there's this white line next to the scrollbar which I'm not entirely sure if it is intended (but I think it would look better without it, but that's just me):
<img width="126" alt="Bildschirmfoto 2024-07-25 um 20 03 08" src="https://github.com/user-attachments/assets/e9303e54-53ab-4d05-b629-31b1b3557cd7">

Also how do I return to the light theme? I can't uncheck the dark theme in user preferences? Can I undo this in the DB somehow? I don't think so since it's BLOB.
<img width="1061" alt="image" src="https://github.com/user-attachments/assets/6cd66826-6dbf-4ebd-8e26-ab4b6091bf6c">

And this color combination is really awkward. I think you should display the badges in the same orange color like in the navigation:
<img width="71" alt="image" src="https://github.com/user-attachments/assets/e940aab0-8b65-4234-ae3f-2edb414338dd">

**Comment by hanneshal** [comment]
Hi @kersten-lohmeyer 

I'm able to switch between default and dark without a problem.

<img width="420" alt="grafik" src="https://github.com/user-attachments/assets/afea5b1e-be3e-4796-a4ab-dba69a2a3d61">

We will address the problem with the avatar and the suggestion regarding the badge colors in the next release.

Thanks for the feedback.

**Comment by kersten-lohmeyer** [comment]
Thanks. I cleared browser cache und restarted but I still don't have a "Default" theme option. How could I possibly fix this? Before the update to 7.1.12 there was nothing to choose in this dropdown. Nothing preselected, just empty.

Also when logged in with another user who has not selected "Dark" it looks like this:
<img width="1209" alt="image" src="https://github.com/user-attachments/assets/7665161d-9e44-4509-b444-89eaf4a9bf30">

Dark is preselected since being the only option. If I save it gets applied but can't go back.

**Comment by hanneshal** [comment]
Hi,

there should be a registration for the default skin:
<img width="1625" alt="grafik" src="https://github.com/user-attachments/assets/a48b8235-bf0a-4aac-abc0-c30bb919a9a5">

Also the sysconfig `Loader::Agent::DefaultSelectedSkin` should be like this:
<img width="579" alt="grafik" src="https://github.com/user-attachments/assets/029e6266-5135-49a1-b020-5428ebc629c1">


I updated the colors as suggested. 
<img width="409" alt="grafik" src="https://github.com/user-attachments/assets/c7262284-a773-4df9-b69a-178775031a4e">

**Comment by kersten-lohmeyer** [comment]
That worked! Thanks a lot! Also for changing the colors. Did you also change the badge colors in the default skin?

**Comment by hanneshal** [comment]
> That worked! Thanks a lot! Also for changing the colors. Did you also change the badge colors in the default skin?

Hi,
cool. No I just changed it in the dark skin. 

Edit:
If you like, you can modify the following file - until the next release is there:
`var/httpd/htdocs/skins/Agent/dark/css/Core.Default.css`
```css
  .Avatar{
     filter: invert(180%) hue-rotate(180deg);
}
    .toolbar-row-wrapper .toolbar-row ul li a .iconWrapper span.Counter {
      background: var(--primary-color);
   }
```

**Comment by kersten-lohmeyer** [comment]
Would be really nice if it was the same color in the default skin. To be honest, I only noticed this in the dark skin. But now I can't unsee it in the default skin. Can you please change it there too?

**Comment by hanneshal** [comment]
> Would be really nice if it was the same color in the default skin. To be honest, I only noticed this in the dark skin. But now I can't unsee it in the default skin. Can you please change it there too?

We will discuss it in the team. Actually I like the read color. 
But it will be discussed

**Comment by kersten-lohmeyer** [comment]
And the white line? Is that intentional? Maybe to make the scrollbar more visible? Also thanks for the temporary fix of the gravatar.

**Comment by kersten-lohmeyer** [comment]
The gravatar CSS did not fix it for me. I cleared Znuny and also browser cache. But that's not that important.

EDIT: Needs to be:

`.UserAvatar{
     filter: invert(180%) hue-rotate(180deg);
}`

But then the Popup has light color:
<img width="279" alt="image" src="https://github.com/user-attachments/assets/e72c9db3-db77-4112-bd5f-74008f5346d1">

**Comment by hanneshal** [comment]
At the moment I can not tell where the white bar is coming from. 
This may take a while.

You change is correct, there is also another rule for the preferences. But this is ok for now.

```CSS
    .UserAvatar img,
    .Avatar img{
       filter: invert(180%) hue-rotate(180deg);
    }
```

**Comment by kersten-lohmeyer** [comment]
> At the moment I can not tell where the white bar is coming from. This may take a while.
> 
> You change is correct, there is also another rule for the preferences. But this is ok for now.
> 
> ```css
>     .UserAvatar img,
>     .Avatar img{
>        filter: invert(180%) hue-rotate(180deg);
>     }
> ```

Ok, perfect now. Thank you.

**Comment by dennykorsukewitz** [comment]
internal MR !1348

**Comment by dennykorsukewitz** [comment]
Fixed https://github.com/znuny/Znuny/commit/4ada84014f4127636e75e06ffbbc9674bdb18209

**Comment by dennykorsukewitz** [comment]
@hanneshal https://github.com/znuny/Znuny/commit/4ada84014f4127636e75e06ffbbc9674bdb18209#commitcomment-144703302

internal MR !1365

Commit: https://github.com/znuny/Znuny/commit/514dd9c1b3737f14b2dae5d7c58a6770ae7a128d

**Comment by dennykorsukewitz** [comment]
Fixed 514dd9c1b3737f14b2dae5d7c58a6770ae7a128d https://github.com/znuny/Znuny/releases/tag/rel-7_1_3
